modules/claims_api/config/schemas/v2/526.json

```diff
@@ -36,7 +36,7 @@
         "additionalProperties": false,
         "required": ["mailingAddress"],
         "properties": {
-          "currentlyVaEmployee": {
+          "currentVaEmployee": {
             "type": "boolean",
```

Comment:
Probably need to make sure the changes like this are in all the related JSON files (request, submission) in order to avoid having these files saying different things.  Then also recompile the dev swagger docs to reflect it.

Reply:
All set.